components/relevancy/src/schema.rs

```diff
@@ -47,7 +55,52 @@ impl ConnectionInitializer for RelevancyConnectionInitializer {
         Ok(db.execute_batch(SQL)?)
     }
 
-    fn upgrade_from(&self, _db: &Transaction<'_>, version: u32) -> open_database::Result<()> {
-        Err(open_database::Error::IncompatibleVersion(version))
+    fn upgrade_from(&self, tx: &Transaction<'_>, version: u32) -> open_database::Result<()> {
+        match version {
+            // Upgrades 1-12 are missing because we started with version 13, because of a
+            // copy-and-paste error.
```

Comment:
Oops!  I think this was my fault when I was copying the code from `suggest`.  It's not that big of a deal though, we can just start with lucky number 13.